Application/EdFi.Ods.Api.NetCore/Extensions/ApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.Ods.Api.NetCore.Middleware;
```

Comment:
These references need to be sorted out, we are getting a build error

![image](https://user-images.githubusercontent.com/288205/88670994-5eff5c00-d0ab-11ea-883d-02ccf5122179.png)